docs/support/tutorials/conda.md

```diff
@@ -57,6 +55,36 @@ from sources, such as [Spack]. Those are not as widely used as Conda, and often
 require some knowledge of the software build process, but they do compile and
 optimize the software for the particular architecture.
 
+### Conda on parallel file system, such as Lustre in Puhti
+
+Conda environments contain a lot of files, and starting a conda application
+requires reading a number of them. Unfortunately all parallel file systems,
+which are optimized for large number of clients, have a poor single client
+performance. You notice this as a longer initial start up time for conda
+applications, and extra stress on the Lustre metadata server. Therefore, we
+recommend two other methods to try before installing conda on parallel file
+system:
+
+1. Use the existing environments available through the module environment
+    
+    Check if any of the already installed environments for Python or R would
+    be suiteble for your project. Regular users cannot install new packages
+    to the system directories, but they can install additional packages to
+    their own directories, for example with
+    
+    ```
+    pip install --user ...
+    ```
+    
+    for python.
+
+2. Use containers
+    
+    This is a great alternative for developing software locally on a workstation,
+    and then deploying it on other workstation, cluster, or on cloud platforms.
+    Puhti supports singularity containers, which are are just single big
```

Comment:
Should we link to some instructions or do we just leave this for the user to figure out?

Reply:
Links would be nice! Please feel free to add links! Although, let's not postpone publishing this, we can add links a bit later too, if not now. I think Ari-Matti has (?) "how to build singularity containers from scratch" instructions cooking.